### PR TITLE
[CELEBORN-807] Adjust shutdown worker logs in LifecycleManager

### PR DESCRIPTION
**Pull request by onebox-li**
### What changes were proposed in this pull request?
In a long run celeborn cluster,  there are some shutdown workers. Whether it is a new task or an old task, even if the worker is not assigned , it will always log below, seems a little noisy.
ERROR CommitManager: Worker xx shutdown, commit all it's partition location.

### Why are the changes needed?
Ditto

### Does this PR introduce _any_ user-facing change?
shutdown worker logs in LifecycleManager changes

### How was this patch tested?
manually test

### CODE REVIEW

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks! cc @RexXiong @AngersZhuuuu could you also take a look at this pr?

**Review by RexXiong — APPROVED** [review]
LGTM, thanks!

File: client/src/main/scala/org/apache/celeborn/client/WorkerStatusTracker.scala [hunk 1]
```diff
@@ -198,10 +207,11 @@ class WorkerStatusTracker(
...
         }
       }
-
-      logInfo(
-        s"Current excluded workers $excludedWorkers, Current shuttingDown ${shuttingWorkers.asScala.map(
-          _.readableAddress()).mkString("\n")}")
+      if (statusChanged) {
+        logInfo(
+          s"Current excluded workers $excludedWorkers, " +
+            s"Current shutting down workers ${shuttingWorkers.asScala.map(_.readableAddress()).mkString("\n")}")
```
[the remark this answers is not in the document]
Comment (onebox-li) [reply client/src/main/scala/org/apache/celeborn/client/WorkerStatusTracker.scala]: OK, done

**Comment by pan3793** [comment]
@onebox-li thanks, I agree that we should suppress the noisy logs. I have experienced that the client excluded all workers and then stuck, will this change help or make it hard to identify such cases?

File: client/src/main/scala/org/apache/celeborn/client/CommitManager.scala [hunk 2]
```diff
@@ -250,6 +250,10 @@ class CommitManager(appUniqueId: String, val conf: CelebornConf, lifecycleManage
     getCommitHandler(shuffleId).isStageEnd(shuffleId)
   }
 
+  def isStageEndOrInProcess(shuffleId: Int): Boolean = {
```
Comment (waitinfuture) [inline client/src/main/scala/org/apache/celeborn/client/CommitManager.scala:253]: nit: IMO it's unnecessary to add this method.

**Comment by onebox-li** [comment]
> @onebox-li thanks, I agree that we should suppress the noisy logs. I have experienced that the client excluded all workers and then stuck, will this change help or make it hard to identify such cases?

I think it may help a bit, too many same logs seem harder to troubleshoot and drown out other important logs.

File: client/src/main/scala/org/apache/celeborn/client/CommitManager.scala (hunk 2 above)
Comment (onebox-li) [reply client/src/main/scala/org/apache/celeborn/client/CommitManager.scala:253]: i just follow `isStageEnd` usage in `CommitManager`

**Pull request closed without merging**